CHANGELOG.md

```diff
@@ -4,14 +4,22 @@
 
 ### What's Changed
```

Comment:
FWIW, I don't think we should go into such details in user-visible changelog. Basically, we should say: "we changed storage internals; mongodump/mongoexport your data before upgrade, mongorestore/mongoimport after upgrade".

But that's not important right now, I will update changelog before release anyway.

Reply:
Thanks! I'm ok with either option. I was trying to follow a similar structure from one of the past release notes where we also gave quite a lot of context, but it it's not necessary, feel free to cut it.